Changed thread affinity checks to be optional (#1334)
BabylonJS/BabylonNative: CMakeLists.txt

option(BABYLON_NATIVE_BUILD_APPS "Build Babylon Native apps." ${PROJECT_IS_TOP_LEVEL})
option(BABYLON_NATIVE_INSTALL "Include the install target." ${PROJECT_IS_TOP_LEVEL})

> # WARNING: This is an experimental features. Only used it if you can ensure that you application will properly handle thread affinity. 

Reviewer: ```suggestion
# WARNING: This is experimental. Only use it if you can ensure that your application will properly handle thread affinity.
```
